examples/lighting-app/nrfconnect/README.md

```diff
@@ -256,6 +253,12 @@ To use the native shell for setup, complete the following steps:
          $ export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
          $ export GNUARMEMB_TOOLCHAIN_PATH=toolchain-dir
 
+4.  If you have the nRF Connect SDK installed, update the nRF Connect SDK to the most recent supported revision
+    by running the following command (replace _chip-dir_ with the path to your nRF Connect SDK installation directory):
```

Comment:
chip-dir is the path to CHIP repository not nRF Connect SDK installation directory.